src/general/MarkdownToBbCodeConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.RegularExpressions;
+
+/// <summary>
+///   Converts markdown to bbcode for displaying
+/// </summary>
+public static class MarkdownToBbCodeConverter
+{
+    private static readonly Regex MarkdownLink = new(@"\[([^\]]+)\]\((http[^\)]+)\)", RegexOptions.Multiline);
```

Comment:
The later line has a `Compiled` option, why not this?

Reply:
I decided to just put it on the regex that looked the most intensive as these are static properties, these objects will be created when the game starts so I thought it best not to use that extensively. Of course I didn't feel like profiling this or anything...